### PR TITLE
Fix -Wsuggest-attribute=format warnings.

### PR DESCRIPTION
**Pull request by marxin**

### CODE REVIEW

File: modules/mono/editor/bindings_generator.cpp [hunk 1]
```diff
@@ -1280,7 +1280,7 @@ Error BindingsGenerator::generate_glue(const String &p_output_dir) {
 
 		List<InternalCall> &custom_icalls = itype.api_type == ClassDB::API_EDITOR ? editor_custom_icalls : core_custom_icalls;
 
-		OS::get_singleton()->print(String("Generating " + itype.name + "...\n").utf8());
+		OS::get_singleton()->print("%s", String("Generating " + itype.name + "...\n").utf8().get_data());
```
Comment (neikeq) [inline modules/mono/editor/bindings_generator.cpp]: Why is it necessary to do this? Also, if we are going to do this, wouldn't it be better this way?:

``` C++
OS::get_singleton()->print("Generating %s...\n", itype.name.utf8().get_data());
```

Comment (marxin) [reply modules/mono/editor/bindings_generator.cpp]: Agree with that, thanks.

File: modules/mono/editor/bindings_generator.cpp [hunk 2]
```diff
@@ -1280,7 +1280,7 @@ Error BindingsGenerator::generate_glue(const String &p_output_dir) {
 
 		List<InternalCall> &custom_icalls = itype.api_type == ClassDB::API_EDITOR ? editor_custom_icalls : core_custom_icalls;
 
-		OS::get_singleton()->print(String("Generating " + itype.name + "...\n").utf8());
+    OS::get_singleton()->print("Generating %s...\n", itype.name.utf8().get_data());
```
Comment (akien-mga) [inline modules/mono/editor/bindings_generator.cpp]: Indentation issue here.

**Review by akien-mga — APPROVED** [review]
Good to merge once indentation is fixed.

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**